CONTRIBUTING.md

```diff
@@ -6,6 +6,24 @@ Please follow common guidelines for our projects [here](https://github.com/packi
 
 - [List of known issues](https://github.com/packit/packit-service-fedmsg/issues) and in case you need to create a new issue, you can do so [here](https://github.com/packit/packit-service-fedmsg/issues/new).
 
+## Testing
+
+We recommend running the tests in a container. You need to have `podman` or
+`docker` installed for this.
+
+To build the test image run:
+
+    make build-test-image
+
+You can run the test now with:
+
+    make check-in-container
+
+`make check` is also available to run the tests in the environment of your
+choice, but first you'll need to make sure to install all the package and test
+dependencies. For now it is up to the reader to figure out how to do this on
+their system.
+
```

Comment:
@bcrocker15 We discussed this morning that the `install-*-deps` targets have a limited value in their current form. More specifically `install-check-deps` should be able to install not only `pytest` but also any other package and/or test dependency. But we couldn't figure out a way to do this without duplicating things already in `setup.cfg` and the playbooks under `files`.

So we agreed to drop these targets for now and:
1. Recommend running the tests in a container.
2. Document that `podman` or `docker` is needed for `make build-test-image`.
3. Mention `make check` that make it clear that package and test dependencies need to be installed first, and that this is left as an exercise for now.

We might come up with some more meaningful `install-check-deps` later on, but right now this is less as a priority.